### DISCUSSION TITLE
Don't reset map (reset focus) card when updating states

### DISCUSSION THREAD

**Discussion by Mincka** [home-assistant/frontend#8066] [Dashboard]
I use the map card in a panel and I move around (change position and zoom) to follow entities.
However, each time there's an entity update, the map is reset and I have to move it again.
It's quite annoying when all your entities are far away from each other.
The focus should be reset only when the user clicks on the dedicated button.

**Comment by karwosts** [comment]
Related #16666

**Reply by Mincka** [reply]
I don't even remember I opened this discussion 2,5 years ago... This one was about the map *card* and that's interesting to see the `autoFit` option was added [10 months ago](https://github.com/home-assistant/frontend/pull/13228). And at the time of this discussion, I believe that there was a updating method not based on `autoFit` but [this method](https://github.com/home-assistant/frontend/blob/1d13947e71cd5ce9b3d02616dd7b2e32d556238e/src/panels/lovelace/cards/hui-map-card.ts#L346), and it was causing this reset in the card, which was not the case anymore until 10 months ago, someone added an option to restore this "original" behavior. Code archeology is fun. 😂
